docs/decisions/0009-persistence-and-retries-for-events.rst

```diff
@@ -0,0 +1,25 @@
+Persistence and retries for events
+==================================
+
+Status
+------
+
+Pending
+
+Context
+-------
+
+`event-routing-backends` transmits events to configured Learning Record Consumer (LRC) via http protocol in near real-time. A strategy needs to be implemented to handle the case when an LRC's link is down.
+
+Decision
+--------
+
+1. Retry attempts shall be made for all events for a configured limit of time and number of retries.
```

Comment:
[Reminder] Consider the case where there are multiple event destinations for a single event - and only a subset of destinations are not working. The successful destinations will then get duplicates.

One idea is to create a separate celery task per event per recipient (at the Routing layer).

Reply:
Separate celery task per event per recipient will be used.